src/services/publishing/publishing.ts

```diff
@@ -55,47 +55,26 @@ const CREATOR_CONTRACTS: CreatorContract = {
   unknown: "0x4Bf7E4777a8D1b6EdD5F2d9b8582e2817F0B0953",
 };
 
-export const getTitleEscrowCreator = async (wallet: Signer): Promise<TitleEscrowCreator> => {
+export const getConnectedRegistry = async (wallet: Signer): Promise<TradeTrustERC721> => {
```

Comment:
nitpick (non-blocking): 
```suggestion
export const getConnectedTokenRegistryContract = async (wallet: Signer): Promise<TradeTrustERC721> => {
```